src/main/webapp/app/exercises/quiz/manage/short-answer-question/short-answer-question-edit.component.ts

```diff
@@ -701,4 +701,18 @@ export class ShortAnswerQuestionEditComponent implements OnInit, OnChanges, Afte
      * @desc reset the question and calls the parsing method of the markdown editor
      */
     prepareForSave(): void {}
+
+    /**
+     * @function toggleExactMatchCheckbox
+     * @desc Sets the similarity value to 100 if the checkbox was checked or to 90 if it was unchecked
+     * @param checked
+     */
+    toggleExactMatchCheckbox(checked: boolean): void {
+        if (checked) {
```

Comment:
you can simplify the lines by doing `this.question.similarityValue = checked ? 100 : 85`